python/instrumentation/openinference-instrumentation-openai/src/openinference/instrumentation/openai/_request.py

```diff
@@ -76,7 +79,11 @@ def _start_as_current_span(
             record_exception=False,
             set_status_on_exception=False,
         ) as span:
-            yield _WithSpan(span=span, extra_attributes=dict(extra_attributes))
+            yield _WithSpan(
+                span=span,
+                context_attributes=dict(context_attributes),
```

Comment:
We should talk about the priority of these attributes.

Reply:
Agreed that a conversation is needed

Reply:
Chatted on it, the approach makes sense.